### PR TITLE
feat: add filtering pid functionality

### PR DESCRIPTION
**Pull request by crazyones110**
## Description

add filtering pid functionality in `kill-process` extension

## Screencast

image
image

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by pernielsentikaer** [comment]
Thanks for the enhancement 🔥 

do you mind checking it @rolandleth 😊

**Comment by crazyones110** [comment]
@pernielsentikaer Any plan when this pr will be merged ?

File: extensions/kill-process/src/index.tsx [hunk 1]
```diff
@@ -94,8 +95,8 @@ export default function ProcessList() {
           const nameMatches = process.name.toLowerCase().includes(query.toLowerCase());
           const pathMatches =
             process.path?.toLowerCase().match(new RegExp(`.+${query}.*\\.[app|framework|prefpane]`, "ig")) != null;
-
-          return nameMatches || (shouldIncludePaths && pathMatches);
+          const pidMatches = process.id.includes(query);
+          return nameMatches || (shouldIncludePaths && pathMatches) || (shouldIncludePid && pidMatches);
```
Comment (rolandleth) [inline extensions/kill-process/src/index.tsx:99]: Just a quick nitpick, but do leave an empty line before `return`.

**Comment by rolandleth** [comment]
@crazyones110 It was my bad, didn't see the notification. Thanks for improving the extension!

**Review by pernielsentikaer — APPROVED** [review]
Hello 👋

I gave it a quick spin, works as expected, approved 🔥

**Pull request merged**